fall18/p3/main.py

```diff
@@ -0,0 +1,76 @@
+import project 
+
+#Function 1

```

Comment:
write this for them, something like this:

def testGetNumRecords():
    pass # replace this with your code